### PR TITLE
Fe v4 upgrade tool

### PR DESCRIPTION
**Pull request by tjololo**
## Related Issue(s)
- #{issue number}

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/altinn-app-cli/Program.cs [hunk 1]
```diff
@@ -30,6 +31,7 @@ static async Task<int> Main(string[] args)
         var skipProcessUpgradeOption = new Option<bool>(name: "--skip-process-upgrade", description: "Skip process file upgrade", getDefaultValue: () => false);
         var skipAppSettingsUpgradeOption = new Option<bool>(name: "--skip-appsettings-upgrade", description: "Skip appsettings file upgrade", getDefaultValue: () => false);
         var rootCommand = new RootCommand("Command line interface for working with Altinn 3 Applications");
+        rootCommand.AddCommand(FrontendUpgrade.GetUpgradeCommand());
```
[the remark this answers is not in the document]
Comment (bjosttveit) [reply src/altinn-app-cli/Program.cs:34]: Yes I agree 👍

**Review by tjololo — COMMENTED** [review]
Looks good to me. I'm not allowed to approve since I'm the creator of the PR, but I'm not the actual writer of the code

File: src/altinn-app-cli/Program.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (tjololo) [reply src/altinn-app-cli/Program.cs:34]: We can do does changes in a separate PR keeping the change isolated to that

**Review by bjosttveit — APPROVED** [review]
Amazing 😆

**Pull request merged**